test/unittest/gtest_helpers.hpp

```diff
@@ -67,8 +67,8 @@ namespace libcyphal
 
 inline void PrintTo(const Duration duration, std::ostream* os)
 {
-    auto locale = os->imbue(std::locale("en_US"));
-    *os << std::chrono::duration_cast<std::chrono::microseconds>(duration).count() << "_us";
+    auto locale = os->imbue(std::locale(""));
+    *os << std::chrono::duration_cast<std::chrono::microseconds>(duration).count() << " us";
```

Comment:
Previous version failed on my "Estonian only" Ubuntu (b/c there is no "en_US" locale there) ;)
BTW, locale stuff here is just to have more human friendly gtest failure output (like `1'000'000 us` vs `1000000 us` without it).